### PR TITLE
Spool 106/update rails version

### PR DESCRIPTION
**Pull request by ksubrama**
[Build](http://wilson.ci.chef.co/job/chef-server-12-trigger-ad_hoc/116/downstreambuildview/)

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: omnibus/config/projects/chef-server.rb [hunk 1]
```diff
@@ -38,6 +38,7 @@
 dependency "preparation"
 
 # global
+dependency "rb-readline"
```
Comment (tylercloke) [inline omnibus/config/projects/chef-server.rb]: Why was this added?

**Comment by tylercloke** [comment]
So I tested this version of rails and the node version bump with oc_id against supermarket and everything seemed to work. Just wondering on the rb-readline.

File: omnibus/config/projects/chef-server.rb (hunk 1 above)
Comment (ksubrama) [inline omnibus/config/projects/chef-server.rb]: Reaction for https://github.com/chef/omnibus-software/pull/627

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

**Comment by jmink** [comment]
:+1:  Thanks for getting that over the line.

**Comment by tylercloke** [comment]
:+1: Let's build it again after the rebase though.
